libs/template/file.go

```diff
@@ -68,16 +74,17 @@ func (f *copyFile) PersistToDisk() error {
 		return err
 	}
 	defer srcFile.Close()
-	dstFile, err := os.OpenFile(path, os.O_CREATE|os.O_EXCL|os.O_WRONLY, f.perm)
+
+	content, err := io.ReadAll(srcFile)
 	if err != nil {
 		return err
 	}
-	defer dstFile.Close()
-	_, err = io.Copy(dstFile, srcFile)
-	return err
+	return writeFile(f.ctx, path, content, f.perm)
 }
 
 type inMemoryFile struct {
```

Comment:
What about `copyFile`?

Reply:
`copyFile` is gone since we have to read the file into memory anyway. Should not be a big issue as these are text files that are relatively small.